
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Input validation module of NRW-GeÃ¼bt">
      
      
        <meta name="author" content="Gregoire Denay">
      
      
        <link rel="canonical" href="https://nrw-geubt.github.io/geuebt-validate/user_guide/">
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../dev_guide/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.0, mkdocs-material-9.5.25">
    
    
      
        <title>Guide for users - geubt-validate</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.6543a935.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#guide-for-users" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow md-header--lifted" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="geubt-validate" class="md-header__button md-logo" aria-label="geubt-validate" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            geubt-validate
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Guide for users
            
          </span>
        </div>
      </div>
    </div>
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/NRW-GEUBT/geuebt-validate" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
    
      
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Welcome

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Guide for users

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../dev_guide/" class="md-tabs__link">
        
  
    
  
  Guide for developers

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../contributing/" class="md-tabs__link">
        
  
    
  
  Contributing

      </a>
    </li>
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../about/license/" class="md-tabs__link">
          
  
  About

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
    
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


  

<nav class="md-nav md-nav--primary md-nav--lifted md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="geubt-validate" class="md-nav__button md-logo" aria-label="geubt-validate" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    geubt-validate
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/NRW-GEUBT/geuebt-validate" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Welcome
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Guide for users
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Guide for users
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#installation" class="md-nav__link">
    <span class="md-ellipsis">
      Installation
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Installation">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#conda" class="md-nav__link">
    <span class="md-ellipsis">
      Conda
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#run-environment" class="md-nav__link">
    <span class="md-ellipsis">
      Run environment
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#install-module-and-databases" class="md-nav__link">
    <span class="md-ellipsis">
      Install module and databases
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#manually-install-databases" class="md-nav__link">
    <span class="md-ellipsis">
      Manually install databases
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Manually install databases">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#kraken2" class="md-nav__link">
    <span class="md-ellipsis">
      Kraken2
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#busco" class="md-nav__link">
    <span class="md-ellipsis">
      Busco
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#taxonomy" class="md-nav__link">
    <span class="md-ellipsis">
      Taxonomy
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#configuration" class="md-nav__link">
    <span class="md-ellipsis">
      Configuration
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Configuration">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#user-defined-parameters" class="md-nav__link">
    <span class="md-ellipsis">
      User defined parameters
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#optional-parameters" class="md-nav__link">
    <span class="md-ellipsis">
      Optional parameters
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#usage" class="md-nav__link">
    <span class="md-ellipsis">
      Usage
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#input-formats" class="md-nav__link">
    <span class="md-ellipsis">
      Input formats
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Input formats">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#metadata-format" class="md-nav__link">
    <span class="md-ellipsis">
      Metadata format
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#sequence-files" class="md-nav__link">
    <span class="md-ellipsis">
      Sequence files
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#assembly-qc-testing" class="md-nav__link">
    <span class="md-ellipsis">
      Assembly QC testing
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#results" class="md-nav__link">
    <span class="md-ellipsis">
      Results
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Results">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#status-report" class="md-nav__link">
    <span class="md-ellipsis">
      Status report
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#isolate-datasheets" class="md-nav__link">
    <span class="md-ellipsis">
      Isolate datasheets
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#assemblies" class="md-nav__link">
    <span class="md-ellipsis">
      Assemblies
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../dev_guide/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Guide for developers
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../contributing/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Contributing
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    About
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            About
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../about/license/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    License
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../about/changelog/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Release notes
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<h1 id="guide-for-users">Guide for users</h1>
<h2 id="installation">Installation</h2>
<h3 id="conda">Conda</h3>
<p>Install conda from any distribution, i.e. miniconda.
You can follow the setup guide from the <a href="https://bioconda.github.io/">Bioconda team</a>.</p>
<p>We advise installing the mamaba solver in the base environement to speed up
environments creation.</p>
<div class="highlight"><pre><span></span><code><a id="__codelineno-0-1" name="__codelineno-0-1" href="#__codelineno-0-1"></a>conda<span class="w"> </span>install<span class="w"> </span>mamba<span class="w"> </span>-n<span class="w"> </span>base<span class="w"> </span>-c<span class="w"> </span>conda-forge
</code></pre></div>
<h3 id="run-environment">Run environment</h3>
<p>The running environement simply required a recent python version (&gt;= 3.9) and snakemake.
If you followed the steps above just run:</p>
<div class="highlight"><pre><span></span><code><a id="__codelineno-1-1" name="__codelineno-1-1" href="#__codelineno-1-1"></a>mamba<span class="w"> </span>create<span class="w"> </span>-n<span class="w"> </span>snakemake<span class="w"> </span>snakemake
</code></pre></div>
<h3 id="install-module-and-databases">Install module and databases</h3>
<p>Download the <a href="https://github.com/NRW-GEUBT/geuebt-validate/releases/latest">latest realease</a>
and unpack it.</p>
<p>If you're feeling brave, clone the repository form Github:</p>
<div class="highlight"><pre><span></span><code><a id="__codelineno-2-1" name="__codelineno-2-1" href="#__codelineno-2-1"></a>git<span class="w"> </span>clone<span class="w"> </span>https://github.com/NRW-GEUBT/geuebt-validate
</code></pre></div>
<p>All software and databases dependencies will be installed during the first run.
It may take a little time so be patient!</p>
<h3 id="manually-install-databases">Manually install databases</h3>
<p>If for any reason you want to manually install database it is possible to provide
a custom path in the configuration.</p>
<p>Following databases are required:</p>
<h4 id="kraken2">Kraken2</h4>
<p>Preformatted kraken2 databases are available from:
<a href="https://benlangmead.github.io/aws-indexes/k2">https://benlangmead.github.io/aws-indexes/k2</a></p>
<p>The workflow uses the Standard-8 (2023-03-14) as default.</p>
<h4 id="busco">Busco</h4>
<p>The workflow uses the bacteria_odb10 for BUSCO.
If you wish to install an alternatve database, see the <a href="https://busco.ezlab.org/busco_userguide.html#offline">user's guide</a>.</p>
<h4 id="taxonomy">Taxonomy</h4>
<p>TaxidTools relies on the local availability of the Taxdump files which can be
obtained from teh <a href="https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/new_taxdump/">NCBI servers</a>.</p>
<p>Using other taxonomy definitions is not supported.</p>
<h2 id="configuration">Configuration</h2>
<p>The configuaration can be defined in two ways:</p>
<ul>
<li>
<p>either edit and locally save the <code>config/config.yaml</code> files and provide its path
  to the snakemake command with the <code>--configfile</code> argument</p>
</li>
<li>
<p>or provide the parameters directly to the snakemake command with
  <code>--config &lt;ARGNAME&gt;=&lt;VALUE&gt;</code></p>
</li>
</ul>
<h3 id="user-defined-parameters">User defined parameters</h3>
<p>Following arguments must be provided for each run:</p>
<table>
<thead>
<tr>
<th>Parameter</th>
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>workdir</code></td>
<td>path-like string</td>
<td>Path to the ouptut directory</td>
</tr>
<tr>
<td><code>metadata</code></td>
<td>path-like string</td>
<td>Path to the metadata file in TSV format</td>
</tr>
<tr>
<td><code>fasta_dir</code></td>
<td>path-like string</td>
<td>Path to the folder containing assembly files in fasta format</td>
</tr>
</tbody>
</table>
<h3 id="optional-parameters">Optional parameters</h3>
<p>Following parameters are optional and will revert to defaults if not set:</p>
<table>
<thead>
<tr>
<th>Parameter</th>
<th>Type</th>
<th>Default</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>max_threads_per_job</code></td>
<td>integer</td>
<td>1</td>
<td>Max number of threads assigned to a single job</td>
</tr>
<tr>
<td><code>kraken2_db</code></td>
<td>path-like string</td>
<td>Default installation in <code>~/.nrw-geuebt/</code></td>
<td>Path to the kraken2 database folder</td>
</tr>
<tr>
<td><code>busco_db</code></td>
<td>path-like string</td>
<td>Default installation in <code>~/.nrw-geuebt/</code></td>
<td>Path to the busco database folder</td>
</tr>
<tr>
<td><code>taxdump</code></td>
<td>path-like string</td>
<td>Default installation in <code>~/.nrw-geuebt/</code></td>
<td>Path to the taxdump database folder</td>
</tr>
<tr>
<td><code>min_contig_length</code></td>
<td>integer</td>
<td>500</td>
<td>Minimal contig length (in bp)<br>to be included in the calculation of assembly<br>statistics</td>
</tr>
</tbody>
</table>
<h2 id="usage">Usage</h2>
<p>The workflow can be started with:</p>
<div class="highlight"><pre><span></span><code><a id="__codelineno-3-1" name="__codelineno-3-1" href="#__codelineno-3-1"></a>snakemake<span class="w"> </span>--use-conda<span class="w"> </span>--conda-prefix<span class="w"> </span>&lt;PATH<span class="w"> </span>TO<span class="w"> </span>CONDA<span class="w"> </span>ENVS&gt;<span class="w"> </span>--configfile<span class="w"> </span>&lt;PATH<span class="w"> </span>TO<span class="w"> </span>CONFIG&gt;<span class="w"> </span>--cores<span class="w"> </span>&lt;N&gt;
</code></pre></div>
<p>See the <a href="https://snakemake.readthedocs.io/en/stable/executing/cli.html">snakemake documentaiton</a> for more information.</p>
<h2 id="input-formats">Input formats</h2>
<h3 id="metadata-format">Metadata format</h3>
<p>The metadata must be provided as a tab-delimited text data, with strict requirements:</p>
<table>
<thead>
<tr>
<th>Field name</th>
<th>Definition</th>
<th>Required?</th>
<th>Allowed Values</th>
</tr>
</thead>
<tbody>
<tr>
<td>isolate_id</td>
<td>Eindeutiger Bezeichner der Isolat</td>
<td>required</td>
<td>free text</td>
</tr>
<tr>
<td>sample_id</td>
<td>Eindeutiger Bezeichner der Probe</td>
<td>required</td>
<td>free text</td>
</tr>
<tr>
<td>alt_isolate_id</td>
<td>Alternativer Bezeichnung der Siolat (z.B. Laborrinternenummer)</td>
<td>optional</td>
<td>free text</td>
</tr>
<tr>
<td>organism</td>
<td>Erwarteter Organismus</td>
<td>required</td>
<td>oneof "Listeria monocytogenes","Salmonella enterica", "Escherichia coli", "Campylobacter spp."</td>
</tr>
<tr>
<td>isolation_org</td>
<td>Identifizierungsname des fÃ¼r die Isolation verantwortlichen Labor</td>
<td>required</td>
<td>oneof "OWL", "RRW", "WFL", "RLD","MEL", "other"</td>
</tr>
<tr>
<td>sequencing_org</td>
<td>Identifizierungsname des fÃ¼r die Sequenzierung verantwortlichen Labor</td>
<td>required</td>
<td>oneof "OWL", "RRW", "WFL", "RLD","MEL", "other"</td>
</tr>
<tr>
<td>bioinformatics_org</td>
<td>Identifizierungsname des fÃ¼r die PrimÃ¤r Analyse verantwortlichen Labor</td>
<td>required</td>
<td>oneof "OWL", "RRW", "WFL", "RLD","MEL", "other"</td>
</tr>
<tr>
<td>third_party_owner</td>
<td>Ggfs. Name der DritteigentÃ¼mer (z.B. bei Humanem Isolaten oder RV)</td>
<td>optional</td>
<td>free text</td>
</tr>
<tr>
<td>extraction_method</td>
<td>Name des Kits oder Methode fÃ¼r die DNA Extraktion</td>
<td>optional</td>
<td>free text</td>
</tr>
<tr>
<td>library_kit</td>
<td>Name des Kits fÃ¼r die DNA-Library Erstellung</td>
<td>optional</td>
<td>free text</td>
</tr>
<tr>
<td>sequencing_kit</td>
<td>Name des Kits fÃ¼r die DNA Sequenzierung</td>
<td>optional</td>
<td>free text</td>
</tr>
<tr>
<td>sequencing_instrument</td>
<td>Model des GerÃ¤t fÃ¼r die DNA Sequenzierung</td>
<td>optional</td>
<td>free text</td>
</tr>
<tr>
<td>assembly_method</td>
<td>Name und Version des verwendeten Software oder Pipeline fÃ¼r die Assembly</td>
<td>optional</td>
<td>free text</td>
</tr>
<tr>
<td>sample_type</td>
<td>Art der Probe (z.B. Lebensmittel, Umfeldprobe, veterinÃ¤rmedizinisches Material, usw.)</td>
<td>required</td>
<td>oneof "Lebensmittel", "Futtermittel", "Umfeld", "Tiergesundheit", "Human", "unknown"</td>
</tr>
<tr>
<td>fasta_name</td>
<td>Eindeutiger Bezeichner der Sequenzdaten (Dateiname)</td>
<td>required</td>
<td>free text</td>
</tr>
<tr>
<td>fasta_md5</td>
<td>MD5 Checksum der Sequenzdaten</td>
<td>required</td>
<td>MD5</td>
</tr>
<tr>
<td>collection_date</td>
<td>Datum der Probenahme</td>
<td>required</td>
<td>ISOdate</td>
</tr>
<tr>
<td>customer</td>
<td>Einsendende KOB</td>
<td>required</td>
<td>Kodierung (2-5 buchstaben)</td>
</tr>
<tr>
<td>manufacturer</td>
<td>Hersteller der Probe</td>
<td>required</td>
<td>Freitext ORÂ  "unknown"</td>
</tr>
<tr>
<td>collection_place</td>
<td>Entnahmeort der Probe</td>
<td>required</td>
<td>AVV / ADV Text</td>
</tr>
<tr>
<td>collection_place_code</td>
<td>Entnahmeort der Probe</td>
<td>required</td>
<td>AVV / ADV Code</td>
</tr>
<tr>
<td>description</td>
<td>Bezeichnung laut Probeentnahmeschein bzw. bei Tupfern Bezeichnung der Entnahmeortes und ggfs. Tierart</td>
<td>required</td>
<td>free text</td>
</tr>
<tr>
<td>manufacturer_type</td>
<td>Einordnung des Entnahmeortes nach AVV-Katalog</td>
<td>optional</td>
<td>AVV Text</td>
</tr>
<tr>
<td>manufacturer_type_code</td>
<td>Einordnung des Entnahmeortes nach AVV-Katalog</td>
<td>optional</td>
<td>AVV Code</td>
</tr>
<tr>
<td>matrix</td>
<td>Einordnung der Probenmatrix nach AVV-Katalog</td>
<td>optional</td>
<td>AVV/TSN Text</td>
</tr>
<tr>
<td>matrix_code</td>
<td>Einordnung der Probenmatrix nach AVV-Katalog</td>
<td>optional</td>
<td>AVV/TSN Code</td>
</tr>
<tr>
<td>collection_cause</td>
<td>Grund fÃ¼r die Probenahme nach AVV-Katalog</td>
<td>optional</td>
<td>AVV Text oder Freitext</td>
</tr>
<tr>
<td>collection_cause_code</td>
<td>Grund fÃ¼r die Probenahme nach AVV-Katalog</td>
<td>optional</td>
<td>AVV Code</td>
</tr>
<tr>
<td>lot_number</td>
<td>Los-Nr./Chargen-Nr. des Herstellers</td>
<td>optional</td>
<td>free text</td>
</tr>
<tr>
<td>seq_depth</td>
<td>Durchschnittliche Sequenzierungstiefe nach Assembly</td>
<td>required</td>
<td>integer</td>
</tr>
<tr>
<td>ref_coverage</td>
<td>Antei Abdeckung des Referenzgenoms</td>
<td>required</td>
<td>float [0-1]</td>
</tr>
<tr>
<td>q30</td>
<td>Anteil der Basen mit PHRED-Score mind. 30</td>
<td>required</td>
<td>float [0-1]</td>
</tr>
</tbody>
</table>
<p>See an example of metadata table in the <code>config</code> folder.</p>
<h3 id="sequence-files">Sequence files</h3>
<p>The assemblies must be provided as fasta file.
Wrapped and unwrapped fastas as well as multifastas are allowed.
There are no special requirements for sequence headers.</p>
<h2 id="assembly-qc-testing">Assembly QC testing</h2>
<p>Assemblies are tested to ensure the quality of input data.
Acceptance criteria are defined by expected species as follows:</p>
<table>
<thead>
<tr>
<th>Parameter</th>
<th>Listeria monocytogenes</th>
<th>Salmonella enterica</th>
<th>Capylobacter spp.</th>
<th>Escherichia coli</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sequencing depth</td>
<td>20 - 200</td>
<td>30 - 200</td>
<td>20 - 200</td>
<td>40 - 200</td>
</tr>
<tr>
<td>Q30</td>
<td>&gt;= 0.80</td>
<td>&gt;= 0.80</td>
<td>&gt;= 0.80</td>
<td>&gt;= 0.80</td>
</tr>
<tr>
<td>Number of contigs &gt; 1 kb</td>
<td>300</td>
<td>300</td>
<td>300</td>
<td>500</td>
</tr>
<tr>
<td>Assembly size (bp)</td>
<td>2,700,000 - 3,200,000</td>
<td>4,300,000 - 5,200,000</td>
<td>1,500,000 - 1,900,000</td>
<td>4,500,000 - 5,900,000</td>
</tr>
<tr>
<td>%GC</td>
<td>33.9 - 41.9</td>
<td>48.1 - 56.1</td>
<td>26.4 - 35.3</td>
<td>46.6 - 54.6</td>
</tr>
<tr>
<td>Orthologs found (%)</td>
<td>&gt;=95 %</td>
<td>&gt;=95 %</td>
<td>&gt;=80%</td>
<td>&gt;=95 %</td>
</tr>
<tr>
<td>Duplicated orthologs (%)</td>
<td>&lt;= 5 %</td>
<td>&lt;= 5 %</td>
<td>&lt;= 5 %</td>
<td>&lt;= 5 %</td>
</tr>
<tr>
<td>Fraction majority Genus</td>
<td>&gt;= 0.95</td>
<td>&gt;= 0.95</td>
<td>&gt;= 0.90</td>
<td>&gt;= 0.90</td>
</tr>
<tr>
<td>Majority genus</td>
<td>Listeria</td>
<td>Salmonella</td>
<td>Campylobacter</td>
<td>Escherichia or Shigella</td>
</tr>
</tbody>
</table>
<h2 id="results">Results</h2>
<p>Results to be used for the next steps are located in the <code>staging</code> folder in the workdir.</p>
<h3 id="status-report">Status report</h3>
<p>A JSON report of the QC checks in the form</p>
<div class="highlight"><pre><span></span><code><a id="__codelineno-4-1" name="__codelineno-4-1" href="#__codelineno-4-1"></a><span class="p">{</span>
<a id="__codelineno-4-2" name="__codelineno-4-2" href="#__codelineno-4-2"></a><span class="w">    </span><span class="nt">&quot;2022-0232977-01&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-4-3" name="__codelineno-4-3" href="#__codelineno-4-3"></a><span class="w">        </span><span class="nt">&quot;STATUS&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;PASS&quot;</span><span class="p">,</span>
<a id="__codelineno-4-4" name="__codelineno-4-4" href="#__codelineno-4-4"></a><span class="w">        </span><span class="nt">&quot;MESSAGES&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">[]</span>
<a id="__codelineno-4-5" name="__codelineno-4-5" href="#__codelineno-4-5"></a><span class="w">    </span><span class="p">},</span>
<a id="__codelineno-4-6" name="__codelineno-4-6" href="#__codelineno-4-6"></a><span class="w">    </span><span class="nt">&quot;2022-0232977-02&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-4-7" name="__codelineno-4-7" href="#__codelineno-4-7"></a><span class="w">        </span><span class="nt">&quot;STATUS&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;FAIL&quot;</span><span class="p">,</span>
<a id="__codelineno-4-8" name="__codelineno-4-8" href="#__codelineno-4-8"></a><span class="w">        </span><span class="nt">&quot;MESSAGES&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">[</span>
<a id="__codelineno-4-9" name="__codelineno-4-9" href="#__codelineno-4-9"></a><span class="w">            </span><span class="s2">&quot;Message checksum&quot;</span><span class="p">,</span>
<a id="__codelineno-4-10" name="__codelineno-4-10" href="#__codelineno-4-10"></a><span class="w">            </span><span class="s2">&quot;Error message assembly&quot;</span>
<a id="__codelineno-4-11" name="__codelineno-4-11" href="#__codelineno-4-11"></a><span class="w">        </span><span class="p">]</span>
<a id="__codelineno-4-12" name="__codelineno-4-12" href="#__codelineno-4-12"></a><span class="w">    </span><span class="p">},</span>
<a id="__codelineno-4-13" name="__codelineno-4-13" href="#__codelineno-4-13"></a><span class="w">    </span><span class="nt">&quot;2022-0232977-03&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-4-14" name="__codelineno-4-14" href="#__codelineno-4-14"></a><span class="w">        </span><span class="nt">&quot;STATUS&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;FAIL&quot;</span><span class="p">,</span>
<a id="__codelineno-4-15" name="__codelineno-4-15" href="#__codelineno-4-15"></a><span class="w">        </span><span class="nt">&quot;MESSAGES&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="s2">&quot;Error message checksum&quot;</span><span class="p">]</span>
<a id="__codelineno-4-16" name="__codelineno-4-16" href="#__codelineno-4-16"></a><span class="w">    </span><span class="p">},</span>
<a id="__codelineno-4-17" name="__codelineno-4-17" href="#__codelineno-4-17"></a><span class="w">    </span><span class="nt">&quot;2022-0232977-04&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-4-18" name="__codelineno-4-18" href="#__codelineno-4-18"></a><span class="w">        </span><span class="nt">&quot;STATUS&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;FAIL&quot;</span><span class="p">,</span>
<a id="__codelineno-4-19" name="__codelineno-4-19" href="#__codelineno-4-19"></a><span class="w">        </span><span class="nt">&quot;MESSAGES&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">[</span>
<a id="__codelineno-4-20" name="__codelineno-4-20" href="#__codelineno-4-20"></a><span class="w">            </span><span class="s2">&quot;Error message metadata 1&quot;</span><span class="p">,</span>
<a id="__codelineno-4-21" name="__codelineno-4-21" href="#__codelineno-4-21"></a><span class="w">            </span><span class="s2">&quot;Error message metadata 2&quot;</span>
<a id="__codelineno-4-22" name="__codelineno-4-22" href="#__codelineno-4-22"></a><span class="w">        </span><span class="p">]</span>
<a id="__codelineno-4-23" name="__codelineno-4-23" href="#__codelineno-4-23"></a><span class="w">    </span><span class="p">}</span>
<a id="__codelineno-4-24" name="__codelineno-4-24" href="#__codelineno-4-24"></a><span class="p">}</span>
</code></pre></div>
<h3 id="isolate-datasheets">Isolate datasheets</h3>
<p>Isolate information are summarized in single JSON files as well as in a merged JSON.
Note that these are generated only for samples satisfying all filters.
The follow the same structure, here for a single entry:</p>
<div class="highlight"><pre><span></span><code><a id="__codelineno-5-1" name="__codelineno-5-1" href="#__codelineno-5-1"></a><span class="p">{</span>
<a id="__codelineno-5-2" name="__codelineno-5-2" href="#__codelineno-5-2"></a><span class="w">    </span><span class="nt">&quot;isolate_id&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;2022-0232977-01&quot;</span><span class="p">,</span>
<a id="__codelineno-5-3" name="__codelineno-5-3" href="#__codelineno-5-3"></a><span class="w">    </span><span class="nt">&quot;sample_id&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;2022-0232977&quot;</span><span class="p">,</span>
<a id="__codelineno-5-4" name="__codelineno-5-4" href="#__codelineno-5-4"></a><span class="w">    </span><span class="nt">&quot;organism&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Salmonella enterica&quot;</span><span class="p">,</span>
<a id="__codelineno-5-5" name="__codelineno-5-5" href="#__codelineno-5-5"></a><span class="w">    </span><span class="nt">&quot;isolate_name_alt&quot;</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span>
<a id="__codelineno-5-6" name="__codelineno-5-6" href="#__codelineno-5-6"></a><span class="w">    </span><span class="nt">&quot;third_party_flag&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;true&quot;</span><span class="p">,</span>
<a id="__codelineno-5-7" name="__codelineno-5-7" href="#__codelineno-5-7"></a><span class="w">    </span><span class="nt">&quot;third_party_owner&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;BfR&quot;</span><span class="p">,</span>
<a id="__codelineno-5-8" name="__codelineno-5-8" href="#__codelineno-5-8"></a><span class="w">    </span><span class="nt">&quot;sample_type&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;unknown&quot;</span><span class="p">,</span>
<a id="__codelineno-5-9" name="__codelineno-5-9" href="#__codelineno-5-9"></a><span class="w">    </span><span class="nt">&quot;fasta_name&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;0232977-01.fasta&quot;</span><span class="p">,</span>
<a id="__codelineno-5-10" name="__codelineno-5-10" href="#__codelineno-5-10"></a><span class="w">    </span><span class="nt">&quot;fasta_md5&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;b9621a5adb8e59cbea8034bfc04fca44&quot;</span><span class="p">,</span>
<a id="__codelineno-5-11" name="__codelineno-5-11" href="#__codelineno-5-11"></a><span class="w">    </span><span class="nt">&quot;isolation&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-5-12" name="__codelineno-5-12" href="#__codelineno-5-12"></a><span class="w">        </span><span class="nt">&quot;org_name&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;unknown&quot;</span>
<a id="__codelineno-5-13" name="__codelineno-5-13" href="#__codelineno-5-13"></a><span class="w">    </span><span class="p">},</span>
<a id="__codelineno-5-14" name="__codelineno-5-14" href="#__codelineno-5-14"></a><span class="w">    </span><span class="nt">&quot;sequencing&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-5-15" name="__codelineno-5-15" href="#__codelineno-5-15"></a><span class="w">        </span><span class="nt">&quot;org_name&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;RRW&quot;</span><span class="p">,</span>
<a id="__codelineno-5-16" name="__codelineno-5-16" href="#__codelineno-5-16"></a><span class="w">        </span><span class="nt">&quot;extraction_method&quot;</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span>
<a id="__codelineno-5-17" name="__codelineno-5-17" href="#__codelineno-5-17"></a><span class="w">        </span><span class="nt">&quot;library_method&quot;</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span>
<a id="__codelineno-5-18" name="__codelineno-5-18" href="#__codelineno-5-18"></a><span class="w">        </span><span class="nt">&quot;sequencing_instrument&quot;</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span>
<a id="__codelineno-5-19" name="__codelineno-5-19" href="#__codelineno-5-19"></a><span class="w">    </span><span class="p">},</span>
<a id="__codelineno-5-20" name="__codelineno-5-20" href="#__codelineno-5-20"></a><span class="w">    </span><span class="nt">&quot;bioinformatics&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-5-21" name="__codelineno-5-21" href="#__codelineno-5-21"></a><span class="w">        </span><span class="nt">&quot;org_name&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;RRW&quot;</span><span class="p">,</span>
<a id="__codelineno-5-22" name="__codelineno-5-22" href="#__codelineno-5-22"></a><span class="w">        </span><span class="nt">&quot;assembly_method&quot;</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span>
<a id="__codelineno-5-23" name="__codelineno-5-23" href="#__codelineno-5-23"></a><span class="w">    </span><span class="p">},</span>
<a id="__codelineno-5-24" name="__codelineno-5-24" href="#__codelineno-5-24"></a><span class="w">    </span><span class="nt">&quot;epidata&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-5-25" name="__codelineno-5-25" href="#__codelineno-5-25"></a><span class="w">        </span><span class="nt">&quot;collection_date&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;2023-03-28&quot;</span><span class="p">,</span>
<a id="__codelineno-5-26" name="__codelineno-5-26" href="#__codelineno-5-26"></a><span class="w">        </span><span class="nt">&quot;municipality&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;unknown&quot;</span><span class="p">,</span>
<a id="__codelineno-5-27" name="__codelineno-5-27" href="#__codelineno-5-27"></a><span class="w">        </span><span class="nt">&quot;country&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;DE&quot;</span><span class="p">,</span>
<a id="__codelineno-5-28" name="__codelineno-5-28" href="#__codelineno-5-28"></a><span class="w">        </span><span class="nt">&quot;cause&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Proficiency Test&quot;</span><span class="p">,</span>
<a id="__codelineno-5-29" name="__codelineno-5-29" href="#__codelineno-5-29"></a><span class="w">        </span><span class="nt">&quot;collected_by&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;unknown&quot;</span><span class="p">,</span>
<a id="__codelineno-5-30" name="__codelineno-5-30" href="#__codelineno-5-30"></a><span class="w">        </span><span class="nt">&quot;manufacturer&quot;</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span>
<a id="__codelineno-5-31" name="__codelineno-5-31" href="#__codelineno-5-31"></a><span class="w">        </span><span class="nt">&quot;designation&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;unknown&quot;</span><span class="p">,</span>
<a id="__codelineno-5-32" name="__codelineno-5-32" href="#__codelineno-5-32"></a><span class="w">        </span><span class="nt">&quot;manufacturer_type&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;unknown&quot;</span><span class="p">,</span>
<a id="__codelineno-5-33" name="__codelineno-5-33" href="#__codelineno-5-33"></a><span class="w">        </span><span class="nt">&quot;sample_description&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;unkown&quot;</span><span class="p">,</span>
<a id="__codelineno-5-34" name="__codelineno-5-34" href="#__codelineno-5-34"></a><span class="w">        </span><span class="nt">&quot;lot_number&quot;</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span>
<a id="__codelineno-5-35" name="__codelineno-5-35" href="#__codelineno-5-35"></a><span class="w">    </span><span class="p">},</span>
<a id="__codelineno-5-36" name="__codelineno-5-36" href="#__codelineno-5-36"></a><span class="w">    </span><span class="nt">&quot;mlst&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-5-37" name="__codelineno-5-37" href="#__codelineno-5-37"></a><span class="w">        </span><span class="nt">&quot;sequence_type&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;34&quot;</span><span class="p">,</span>
<a id="__codelineno-5-38" name="__codelineno-5-38" href="#__codelineno-5-38"></a><span class="w">        </span><span class="nt">&quot;scheme&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;senterica_achtman_2&quot;</span><span class="p">,</span>
<a id="__codelineno-5-39" name="__codelineno-5-39" href="#__codelineno-5-39"></a><span class="w">        </span><span class="nt">&quot;alleles&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-5-40" name="__codelineno-5-40" href="#__codelineno-5-40"></a><span class="w">            </span><span class="nt">&quot;purE&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;5&quot;</span><span class="p">,</span>
<a id="__codelineno-5-41" name="__codelineno-5-41" href="#__codelineno-5-41"></a><span class="w">            </span><span class="nt">&quot;hemD&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;12&quot;</span><span class="p">,</span>
<a id="__codelineno-5-42" name="__codelineno-5-42" href="#__codelineno-5-42"></a><span class="w">            </span><span class="nt">&quot;thrA&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;2&quot;</span><span class="p">,</span>
<a id="__codelineno-5-43" name="__codelineno-5-43" href="#__codelineno-5-43"></a><span class="w">            </span><span class="nt">&quot;sucA&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;9&quot;</span><span class="p">,</span>
<a id="__codelineno-5-44" name="__codelineno-5-44" href="#__codelineno-5-44"></a><span class="w">            </span><span class="nt">&quot;hisD&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;9&quot;</span><span class="p">,</span>
<a id="__codelineno-5-45" name="__codelineno-5-45" href="#__codelineno-5-45"></a><span class="w">            </span><span class="nt">&quot;dnaN&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;19&quot;</span><span class="p">,</span>
<a id="__codelineno-5-46" name="__codelineno-5-46" href="#__codelineno-5-46"></a><span class="w">            </span><span class="nt">&quot;aroC&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;10&quot;</span>
<a id="__codelineno-5-47" name="__codelineno-5-47" href="#__codelineno-5-47"></a><span class="w">        </span><span class="p">}</span>
<a id="__codelineno-5-48" name="__codelineno-5-48" href="#__codelineno-5-48"></a><span class="w">    </span><span class="p">},</span>
<a id="__codelineno-5-49" name="__codelineno-5-49" href="#__codelineno-5-49"></a><span class="w">    </span><span class="nt">&quot;qc_metrics&quot;</span><span class="p">:</span><span class="w"> </span><span class="p">{</span>
<a id="__codelineno-5-50" name="__codelineno-5-50" href="#__codelineno-5-50"></a><span class="w">        </span><span class="nt">&quot;sequencing_depth&quot;</span><span class="p">:</span><span class="w"> </span><span class="mf">45.6</span><span class="p">,</span>
<a id="__codelineno-5-51" name="__codelineno-5-51" href="#__codelineno-5-51"></a><span class="w">        </span><span class="nt">&quot;ref_coverage&quot;</span><span class="p">:</span><span class="w"> </span><span class="mf">0.99</span><span class="p">,</span>
<a id="__codelineno-5-52" name="__codelineno-5-52" href="#__codelineno-5-52"></a><span class="w">        </span><span class="nt">&quot;q30&quot;</span><span class="p">:</span><span class="w"> </span><span class="mf">0.9</span><span class="p">,</span>
<a id="__codelineno-5-53" name="__codelineno-5-53" href="#__codelineno-5-53"></a><span class="w">        </span><span class="nt">&quot;N50&quot;</span><span class="p">:</span><span class="w"> </span><span class="mi">284333</span><span class="p">,</span>
<a id="__codelineno-5-54" name="__codelineno-5-54" href="#__codelineno-5-54"></a><span class="w">        </span><span class="nt">&quot;L50&quot;</span><span class="p">:</span><span class="w"> </span><span class="mi">5</span><span class="p">,</span>
<a id="__codelineno-5-55" name="__codelineno-5-55" href="#__codelineno-5-55"></a><span class="w">        </span><span class="nt">&quot;n_contigs_1kbp&quot;</span><span class="p">:</span><span class="w"> </span><span class="mi">53</span><span class="p">,</span>
<a id="__codelineno-5-56" name="__codelineno-5-56" href="#__codelineno-5-56"></a><span class="w">        </span><span class="nt">&quot;assembly_size&quot;</span><span class="p">:</span><span class="w"> </span><span class="mi">4960041</span><span class="p">,</span>
<a id="__codelineno-5-57" name="__codelineno-5-57" href="#__codelineno-5-57"></a><span class="w">        </span><span class="nt">&quot;GC_perc&quot;</span><span class="p">:</span><span class="w"> </span><span class="mf">52.13</span><span class="p">,</span>
<a id="__codelineno-5-58" name="__codelineno-5-58" href="#__codelineno-5-58"></a><span class="w">        </span><span class="nt">&quot;orthologs_found&quot;</span><span class="p">:</span><span class="w"> </span><span class="mf">98.4</span><span class="p">,</span>
<a id="__codelineno-5-59" name="__codelineno-5-59" href="#__codelineno-5-59"></a><span class="w">        </span><span class="nt">&quot;duplicated_orthologs&quot;</span><span class="p">:</span><span class="w"> </span><span class="mf">0.0</span><span class="p">,</span>
<a id="__codelineno-5-60" name="__codelineno-5-60" href="#__codelineno-5-60"></a><span class="w">        </span><span class="nt">&quot;majority_genus&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Salmonella&quot;</span><span class="p">,</span>
<a id="__codelineno-5-61" name="__codelineno-5-61" href="#__codelineno-5-61"></a><span class="w">        </span><span class="nt">&quot;fraction_majority_genus&quot;</span><span class="p">:</span><span class="w"> </span><span class="mf">0.9971668689656122</span><span class="p">,</span>
<a id="__codelineno-5-62" name="__codelineno-5-62" href="#__codelineno-5-62"></a><span class="w">        </span><span class="nt">&quot;majority_species&quot;</span><span class="p">:</span><span class="w"> </span><span class="s2">&quot;Salmonella enterica&quot;</span><span class="p">,</span>
<a id="__codelineno-5-63" name="__codelineno-5-63" href="#__codelineno-5-63"></a><span class="w">        </span><span class="nt">&quot;fraction_majority_species&quot;</span><span class="p">:</span><span class="w"> </span><span class="mf">0.9971575751364787</span>
<a id="__codelineno-5-64" name="__codelineno-5-64" href="#__codelineno-5-64"></a><span class="w">    </span><span class="p">}</span>
<a id="__codelineno-5-65" name="__codelineno-5-65" href="#__codelineno-5-65"></a><span class="p">}</span>
</code></pre></div>
<h3 id="assemblies">Assemblies</h3>
<p>Assemblies are renamed so that the fastas are in the form <code>&lt;ISOLATE ID&gt;.fa</code> and
copied to the <code>staging/fastas/</code>folder of the workdir.</p>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Copyright &copy; 2023 GrÃ©goire Denay - <a href="#__consent">Change cookie settings</a>

    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.instant", "navigation.tracking", "navigation.tabs", "navigation.tabs.sticky", "navigation.sections", "navigation.top", "toc.integrate", "content.code.annotate"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.081f42fc.min.js"></script>
      
        <script src="../javascripts/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
    
  </body>
</html>